src/autora/doc/pipelines/main.py

```diff
@@ -49,7 +49,7 @@ def evaluate_documentation(predictions: List[str], references: List[str]) -> Tup
 
 
 @app.command(help="Evaluate a model for code-to-documentation generation for all prompts in the prompts_file")
-def eval_on_prompts_file(
+def eval_prompts(
```

Comment:
I was about to ask you to add a doc-comment for this function. In particular because it's hard to tell what the `List[Dict[str,str]]` will contain. But I think a better option is to create a type (a [dataclass](https://docs.python.org/3/library/dataclasses.html)?) for the return type, e.g. an `EvalResult` class.